### PR TITLE
Update bylaws.md

### PR DESCRIPTION
**Pull request by yabellini**
- Change monthly meeting to quarterly meeting.
- Adding mention to Parliamentary Procedure.

As discussed in Issue carpentries/2022_Bylaw_revision/issues/11

### CODE REVIEW

File: topic_folders/governance/bylaws.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Executive Council members will not serve for more than three consecutive 2 year
...
 
 ### Regular Meetings and Quorum
-The Executive Council must meet at least ten times per year, either in person or online. Quorum for meetings of the Executive Council is five Members, including at least one Council-elected and one Community-elected Member. The Executive Council may use an asynchronous public online voting mechanism for motions to be approved inbetween regular meetings.
+The Executive Council must meet at least quarterly, either in person or online. The meetings follow the parliamentary procedure documented in the Carpentries Handbook. Quorum for meetings of the Executive Council is five Members, including at least one Council-elected and one Community-elected Member. The Executive Council may use an asynchronous public online voting mechanism for motions to be approved inbetween regular meetings.
```
Comment (yabellini) [inline topic_folders/governance/bylaws.md:85]:
```suggestion
The Executive Council must meet at least quarterly, either in person or online. The meetings follow the [parliamentary procedure documented in the Carpentries Handbook](https://docs.carpentries.org/topic_folders/governance/executive-council.html#meetings). Quorum for meetings of the Executive Council is five Members, including at least one Council-elected and one Community-elected Member. The Executive Council may use an asynchronous public online voting mechanism for motions to be approved inbetween regular meetings.
```

**Comment by yabellini** [comment]
Done. Please take into account that we need to update the Handbook with the new parliamentary procedures.

**Review by kariljordan — APPROVED** [review]
This is perfect. Thank you @yabellini! 

This needs to be approved by the Executive Council before it is merged into the bylaws.

**Comment by sstevens2** [comment]
A motion regarding this change was passed in the 2021 December meeting.
> Voting: I move to adopt a Quarterly meeting schedule as described in the proposal linked to above, starting with the 2022 Executive Council. (Maker: Treasurer Smith) Motion passes (7 yes, 0 no, 1 abstiained).

**Pull request merged**